srcpkgs/rockpro64-base/template

```diff
@@ -0,0 +1,11 @@
+# Template file for 'rockpro64-base'
+pkgname=rockpro64-base
+version=1
+revision=1
+archs="aarch64*"
+build_style=meta
+depends="rockpro64-uboot u-boot-menu dracut"
```

Comment:
The other base templates such as pinephone-base and pinephonepro-base include a kernel in the depends.  Even though the rockpro64 doesn't require a custom kernel we should include `linux` as a dependency so images created with `rockpro64-base` will automatically have a kernel included.
```suggestion
depends="linux rockpro64-uboot u-boot-menu dracut"
```

Reply:
I think this is a good idea.

Reply:
Yep, this did bite me earlier trying to figure out why there is no kernel installed in my image. Took me some time to realize this was just left out from the base-pkg.

Reply:
I think I'll have some time this weekend so if @tiemenwerkman doesn't respond to the above suggestion I can make a new PR.